compiler/src/iree/compiler/Codegen/Utils/GPUUtils.h

```diff
@@ -156,6 +157,11 @@ Value unpackToVector(Location loc, OpBuilder &builder, Value packedInput,
 /// Emit identity constant based on combiningKind and type.
 Value getCombiningIdentityValue(Location loc, OpBuilder &builder,
                                 vector::CombiningKind kind, Type identityType);
+
+/// Returns the matching GPU reduction operation.
+mlir::gpu::AllReduceOperation
+combiningKindToAllReduce(vector::CombiningKind kind);
```

Comment:
Is this used anywhere?

Reply:
Yes, the single use is introduced by this commit.

Reply:
Oh, I've just found it, never mind